sdk/nodejs/Makefile

```diff
@@ -24,14 +24,16 @@ GO_TEST_FAST = $(PYTHON) ../../scripts/go-test.py $(GO_TEST_FAST_FLAGS)
 
 ensure:: yarn.ensure node.ensure .ensure.phony
 .ensure.phony: package.json
-	yarn install --frozen-lockfile
+	yarn install
 	@touch .ensure.phony
 
 lint:: ensure
 	yarn run eslint -c .eslintrc.js --ext .ts .
 
 build_package:: ensure
 	yarn run tsc
+	npx esbuild --bundle --tree-shaking=true --minify --format=cjs --allow-overwrite --platform=node --define:window=global --target=node14 --outdir=bundled_dist cmd/run/index.ts
+	cp bundled_dist/index.js bin/robbies_vendor.js
```

Comment:
robbies_vendor.js?

Reply:
yessir!
It's a draft PR for a reason ;D lol